README.md

```diff
@@ -41,7 +41,7 @@ Advanced usage (with default values):
       tank/example/dataset:
         enabled: true
         frequent: false
-        hourly: false        
+        hourly: false       
```

Comment:
there is still whitespace at the end of the line

Reply:
```suggestion
        hourly: false
```